config/fcos/v1_6_exp/validate.go

```diff
@@ -54,13 +54,12 @@ func (user GrubUser) Validate(c path.ContextPath) (r report.Report) {
 
 func (c Config) Validate() (r report.Report) {
 	for _, disk := range c.Storage.Disks {
-		if disk.Device == "/dev/disk/by-id/coreos-boot-disk" {
-			continue
-		}
-		if disk.WipeTable != nil && !*disk.WipeTable {
-			for p, partition := range disk.Partitions {
-				if partition.Number == 0 {
-					r.AddOnWarn(path.New("partition_number", p), common.ErrInvalidPartitionConfig)
+		if disk.Device != "/dev/disk/by-id/coreos-boot-disk" {
```

Comment:
Hmm I was thinking more along the lines of 
```go
rootDevice := "/dev/disk/by-id/coreos-boot-disk"
if disk.Device != rootDevice && util.IsFalse(disk.WipeTable) {
//do work
}
```

Reply:
I see what you meant now my bad! ill add that in my next push.